src/utils.jl

```diff
@@ -48,13 +48,22 @@ Else return the one and only element.
 function getattr(o::Observable, idx, default=nothing)
     if o[] isa AbstractVector && !isa(o[], Point)
         return o[][idx]
+    elseif o[] isa DefaultDict || o[] isa DefaultOrderedDict
+        return getindex(o[], idx)
     elseif o[] isa AbstractDict
         return get(o[], idx, default)
     else
         return o[] === nothing ? default : o[]
     end
 end
 
+"""
+    isscalar(o::Observable)
+
+Return `true` if `Observable` is not an `AbstractVector` or an `AbstractDict`
+"""
+isscalar(o) = !isa(o, AbstractVector) && !isa(o, AbstractDict)
```

Comment:
>Maybe the name isscalar is a bit misleading

Yes, naming it `issingleattribute` is better. 

> Do we have arguments where people might put a single `Point` ?

good catch. Yes there are the offsets that one can pass in a `Point`
So I guess we should update the function to 
```julia
issingleattribute(x) = isa(x, Point) || (!isa(x, AbstractVector) && !isa(x, AbstractDict))
```
(also substitute `o` with `x` since it's misleading that it might be an Observable while it's not)